src/status_im/data_store/realm/schemas/base/migrations.cljs

```diff
@@ -130,3 +131,42 @@
 
 (defn v18 [old-realm new-realm]
   (log/debug "migrating base database v18: " old-realm new-realm))
+
+;; transform inactive legacy network values into the current ones
+(defn- transition-rpc-url [rpc-url]
```

Comment:
you could use a case statement here instead of `cond` as you are always checking for equality to a constant

Reply:
cool! now it is more concise